### PR TITLE
Cleaned up the usage information

### PR DESCRIPTION
**Pull request by leo**
This PR makes the usage information of all sub commands consistent and removes unnecessary examples to keep it as short as possible (as discussed with @matheuss).

In addition, it makes `now login <email>` work (which didn't before) and adds the usage information for the new `now logout` sub command.

### CODE REVIEW

File: src/providers/sh/commands/certs.js [hunk 1]
```diff
@@ -21,13 +21,17 @@ const logo = require('../../../util/output/logo')
 
 const help = () => {
   console.log(`
-  ${chalk.bold(`${logo} now certs`)} <ls | create | renew | replace | rm> <cn>
+  ${chalk.bold(`${logo} now certs`)} [options] <command>
 
-  ${chalk.dim('Note:')}
+  ${chalk.yellow('NOTE:')} This command is intended for advanced use only. By default,
```
Comment (TooTallNate) [inline src/providers/sh/commands/certs.js]: Maybe put `By default,` combined with the line below.

File: src/providers/sh/commands/dns.js [hunk 2]
```diff
@@ -42,23 +42,26 @@ const help = () => {
...
 
-  ${chalk.gray('–')} List all your DNS records
-
-      ${chalk.cyan('$ now dns ls')}
-
   ${chalk.gray('–')} Add an A record for a subdomain
 
       ${chalk.cyan(
-        '$ now dns add <YOUR DOMAIN> <SUBDOMAIN NAME> A <RECORD VALUE>'
+        '$ now dns add <DOMAIN> <SUBDOMAIN> A | AAAA | ALIAS | CNAME | TXT>  <VALUE>'
```
Comment (TooTallNate) [inline src/providers/sh/commands/dns.js]: There's an extra `>` after TXT (or a missing `<` before A).

Comment (leo) [reply src/providers/sh/commands/dns.js]: ✅

File: src/providers/sh/commands/certs.js (hunk 1 above)
Comment (leo) [reply src/providers/sh/commands/certs.js]: ✅

**Comment by TooTallNate** [comment]
Great stuff @leo! 👍

**Pull request merged**